build/parser.mjs

```
@@ -445,17 +446,25 @@ class Parser {
-    // eslint-disable-next-line no-restricted-syntax
-    for (const type of types) {
-      const contains = type.regex ? new RegExp(type.id, 'ig').test(item.uniqueName) : item.uniqueName.includes(type.id);
-      if (contains) {
-        if (type.append) item.type = `${item.type}${type.name}`;
-        else item.type = type.name;
-        // if (item.type !== type.name) console.error(`${item.name} didn't update types`)
-        break;
+    const arcane = data.wikia.arcanes.find((entry) => entry.name === item.name);
+    if (arcane) {
+      item.type = `${arcane.type} Arcane`;
+    } else if (!arcaneType && item.name) {
```

ESLint

[error] 'arcaneType' is not defined.

(no-undef)